runtime/common/src/lib.rs

```diff
@@ -1322,23 +1347,45 @@ macro_rules! create_runtime_api {
                     impl BaselineConfig for Runtime {}
 
                     let whitelist: Vec<TrackedStorageKey> = vec![
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac")
-                            .to_vec()
-                            .into(),
-                        hex_literal::hex!("c2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80")
-                            .to_vec()
-                            .into(),
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7ff553b5a9862a516939d82b3d3d8661a")
-                            .to_vec()
-                            .into(),
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850")
-                            .to_vec()
-                            .into(),
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7")
-                            .to_vec()
-                            .into(),
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da946c154ffd9992e395af90b5b13cc6f295c77033fce8a9045824a6690bbf99c6db269502f0a8d1d2a008542d5690a0749").to_vec().into(),
-                        hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```

Comment:
Good questions.

`26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da946c154ffd9992e395af90b5b13cc6f295c77033fce8a9045824a6690bbf99c6db269502f0a8d1d2a008542d5690a0749` is `frame_benchmarking::account("caller", 0, 0)`. I removed it because the assumption that this account is read every block anyways and consequently is cached, which in turn means that additional reads and writes to this account can be ignored, is not given. The account simply is not read every block by default.

`26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000` is an invalid treasury account (the account derived from `PalletId(*b"aca/trsy")`)